frontend/app/application.py

```diff
@@ -195,6 +195,11 @@ def project_page_template(project_id):
 
 def get_year_from_date_string(date_string):
     return(date_string[0:4])
+def get_project_schema():
```

Comment:
This may be a duplicate of https://github.com/research-software-directory/research-software-directory/blob/a753e2e8c91fa162445e066657addf1182bbf624/backend/src/routes.py#L275-L278

Also I think the returned schema is schema of everything, therefore the function name is a bit misleading

Reply:
done